shell/platform/darwin/ios/framework/Source/FlutterPlatformViewsTest.mm

```diff
@@ -602,6 +602,57 @@ - (void)testSetFlutterViewControllerAfterCreateCanStillDispatchTouchEvents {
   flutterPlatformViewsController->Reset();
 }
 
+- (void)testFlutterPlatformViewControllerSubmitFrameWithoutFlutterViewNotCrashing {
```

Comment:
This test appears to only have one assert.  Why don't you check the bool value that SubmitFrame returns at least.

Reply:
Done!